AY2021S2-CS2103-T14-1/tp
# Feat(issue): Update find command to search by tags and description

### What this does
- Updates `ifind` command to search within issue tags and description as well
- Updates `ifind` documentation to reflect new behaviour

Closes #94 

### How to test
1. Add an issue to SunRez with a tag of your choosing
2. Run `ifind [whole or partial tag]`
3. Room with that tag should appear. Any room with that keyword in the description should also appear. 
4. Repeat the above with words from the description or room number. You can mix and match these and it should return the room containing any of the keywords present.
